resources/views/categories/index.blade.php

```diff
@@ -0,0 +1,68 @@
+@extends('layouts.master')
+
+@section('main-content')
+    <div class="box">
+        <div class="box-header">
+            <h3 class="box-title">List Categories</h3>
+            <button id="btn-add-category" class="btn btn-primary" href="{{ route('categories.create') }}" style="float: right" title="Add Category">
```

Comment:
don't use inline css. If you want to float right I think Adminlte already have class for it.